tests/remote/integration_google_cloud/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "client_email" {
```

Comment:
isn't there suppose to be a `main.tf` in here too?  My understanding is there should be an instance of the actual gcp integration here - right @davidbloss ?

Reply:
Correct, running `task test-integration -- "-filter=integration_google_cloud.tftest.hcl"` locally on this branch gave me
```tf
integration_google_cloud.tftest.hcl... in progress
  run "resource_integration_google_cloud_with_ownership_tag_keys"... fail
╷
│ Error: Reference to undeclared resource
│
│   on integration_google_cloud/outputs.tf line 2, in output "test":
│    2:   value     = opslevel_integration_google_cloud.test
│
│ A managed resource "opslevel_integration_google_cloud" "test" has not been declared in the root module.
╵
  run "resource_integration_google_cloud_without_ownership_tag_keys"... skip
integration_google_cloud.tftest.hcl... tearing down
integration_google_cloud.tftest.hcl... fail

Failure! 0 passed, 1 failed, 1 skipped.
```

Reply:
```
~/repos/terraform-provider-opslevel ta/gcp-1 !2 ❯ git add tests/remote/integration_google_cloud/main.tf                                                                                              Ruby 3.0.0 04:50:04 PM
The following paths are ignored by one of your .gitignore files:
tests/remote/integration_google_cloud/main.tf
hint: Use -f if you really want to add them.
hint: Disable this message with "git config advice.addIgnoredFile false"

```

This gitignore rule is preventing it from being added without `add -f` - https://github.com/OpsLevel/terraform-provider-opslevel/blob/3e62a0c875e4cc96ebb1de910748cf75c714c03c/.gitignore#L47

Committed the file

Reply:
Same with backend.tf